default/stringtables/ru.txt

```diff
@@ -952,6 +988,178 @@ ERROR_TOO_MANY_HUMAN_PLAYERS
+
+RULE_SHIP_STRUCTURE_FACTOR
+Броня кораблей
+
+RULE_SHIP_STRUCTURE_FACTOR_DESC
+Регулятор параметров защиты для корпусов кораблей и деталей их брони.
+
+RULE_SHIP_HULL_COST_FACTOR
+Топливо кораблей
+
+RULE_SHIP_HULL_COST_FACTOR_DESC
+Регулятор затрат топлива у кораблей разных типов.
+
+RULE_SHIP_PART_COST_FACTOR
+Стоимость деталей
+
+RULE_SHIP_PART_COST_FACTOR_DESC
+Регулятор стоимости отдельных деталей кораблей.
+
+RULE_BUILDING_COST_FACTOR
+Стоимость строений
+
+RULE_BUILDING_COST_FACTOR_DESC
+Регулятор стоимости различных строений.
+
+RULE_TECH_COST_FACTOR
+Стоимость исследований
+
+RULE_TECH_COST_FACTOR_DESC
+Регулятор затрат на изучение различных наук.
+
+RULE_ENABLE_EXPERIMENTORS
+Генерация Экспериментаторов
+
+RULE_ENABLE_EXPERIMENTORS_DESC
+Создает планету с Экспериментаторами во время генерации галактики.
+
+RULE_ENABLE_SUPER_TESTER
+Супер-тестерское поглощение
+
+RULE_ENABLE_SUPER_TESTER_DESC
+Создаёт возможность строительства Супер-тестерского поглощения.
+
+RULE_STOCKPILE_IMPORT_LIMITED
+Предел импорта резервов
+
+RULE_STOCKPILE_IMPORT_LIMITED_DESC
+Активирует предел для импорта резервов за ход.
+
+RULE_STARLANES_EVERYWHERE
+Звёздные пути повсюду
+
+RULE_STARLANES_EVERYWHERE_DESC
+Звёздные пути образуются автоматически между любыми двумя ближайшими системами, вне зависимости от первоначальной формы галактики. Они также не отображаются на галактической карте, но всё же могут быть удалены в ходе игрового процесса.
+
+RULE_HABITABLE_SIZE_TINY
+Население крошечных планет
+
+RULE_HABITABLE_SIZE_SMALL
+Население маленьких планет
+
+RULE_HABITABLE_SIZE_MEDIUM
+Население средних планет
+
+RULE_HABITABLE_SIZE_LARGE
+Население больших планет
+
+RULE_HABITABLE_SIZE_HUGE
+Население огромных планет
+
+RULE_HABITABLE_SIZE_ASTEROIDS
+Население астероидов
+
+RULE_HABITABLE_SIZE_GASGIANT
+Население газовых гигантов
+
+RULE_HABITABLE_SIZE_DESC
+Значение, задающее максимальное число параметра [[encyclopedia POPULATION_TITLE_SHORT_DESC]] для планет данного размера.
+
+RULE_ALLOW_CONCEDE
+Разрешить сдаваться
+
+RULE_ALLOW_CONCEDE_DESC
+Разрешает империям сдаваться. При этом уничтожаются любые данные об этих империях. Далее игра продолжается среди оставшихся игроков.
+
+RULE_CONCEDE_COLONIES_THRESHOLD
+MAX колоний для поражения
+
+RULE_CONCEDE_COLONIES_THRESHOLD_DESC
+Империи, имеющие большее число колоний, чем это число, не могут сдаваться.
+
+RULE_THRESHOLD_HUMAN_PLAYER_WIN
+Максимум победителей
+
+RULE_THRESHOLD_HUMAN_PLAYER_WIN_DESC
+Максимальное число игроков-людей, которые могут победить в мультиплеере, когда в галактике больше не осталось других выживших игроков-людей.
+
+RULE_ONLY_ALLIANCE_WIN
+Побеждают только альянсы
+
+RULE_ONLY_ALLIANCE_WIN_DESC
+Игроки в альянсе могут побеждать вместе, если число человеческих игроков в их альянсе не превышает возможное количество людей-победителей для данной игры. При этом в галактике не должно оставаться других выживших игроков-людей.
+
+RULE_SHIP_PART_BASED_UPKEEP
```

Comment:
Очень далеко от оригинала.
```
RULE_SHIP_PART_BASED_UPKEEP
Use part-based upkeep

RULE_SHIP_PART_BASED_UPKEEP_DESC
Calculate upkeep based on ship parts instead of ships themselves.
```

Reply:
upkeep это затраты на содержание.  

Хотя самого содержания в игре нет, но есть его эмуляция. Стоимость корабля равна базовая стоимость +  коофициент * количество кораблей (кажется любых).

Эта опция переключает корабли на компоненты. 

@o01eg , можешь рассказать подробнее, что это и как влияет на процесс игры? 
Я так же не очень понял, ести ли это в списке изменений https://github.com/freeorion/freeorion/blob/v0.4.8/ChangeLog.md

Reply:
В 0.4.8 это не вошло. Я взял эту формулу из форка @Voker57 , но он сам её взял то ли с форума, то ли с ещё чьего-то PR.

Reply:
А ты можешь рассказать как это действует?  

Я бы ожидал такое поведение:

```
При включении опции цена каждого компонента зависит от количества таких компонентов во всех в кораблях флота.  
Это заставляет игрока  и Компьютер строить корбали различного типа с разными компонентами.
```

Reply:
Насчёт заставляет Компьютер не уверен, скорее всего ИИ эту опцию не понимает. Я так понимаю, это для того, чтобы бы стимул строить небольшие корабли, без этой опции лучше делать мало кораблей забитых под завязку. Но как оно отражается на балансе я надеюсь @Voker57 ответит, он с этой формлой играл больше.

Reply:
Цена каждого корабля зависит от суммарного количества компонентов в существующих кораблях.

Там еще разные множители для разных типов компонентов, но в этом описании это несущественно.

Reply:
> Давайте тогда уберём из переводов.

Мы тут уже в четвером ведём беседу, если сейчас окончим, значит время было потраченно зря. 

Предлагаю английски вариант обновить до такого:
```
RULE_SHIP_PART_BASED_UPKEEP_DESC
Calculate upkeep based on ship parts count instead of ships count. 
With this option, Scout and BattleShip will cause the different impact on [[encyclopedia FLEET_UPKEEP_TITLE]].
```
 
@Voker57 , можешь дописать статью в энциклопедии?

Reply:
@Cjkjvfnby не понял что куда дописать. Английский вариант который ты процитировал?

Reply:
В статью [[encyclopedia FLEET_UPKEEP_TITLE]]. В английскую.  Можно сразу и в русскую.

То, что я процитировал это предложение. На выходных его в виде пулреквеста оформлю.